### PR TITLE
cleaning garbage

### PR DESCRIPTION
**Pull request by chebbix**

### CODE REVIEW

**Comment by skqr** [comment]
Change that name to something PG-13 or I'll be visiting you tonight and dying your brown so you're no longer special, @chebbix

File: Util/RepositoryHelper.php [hunk 1]
```diff
@@ -79,6 +81,14 @@ public function findPaginated(
             }
         }
 
+        // @todo Esto se podría abstraer en un DefaultSorting que se pueda extender
```
Comment (skqr) [inline Util/RepositoryHelper.php]: Let's keep it in English for the FOSS repos. Do it for the children, @chebbix

File: Util/RepositoryHelper.php [hunk 2]
```diff
@@ -79,6 +81,14 @@ public function findPaginated(
...
+        // @todo Faltaría definir la forma de filtrar por campos de entidades relacionadas
+        foreach($sorting as $entityName => $sort) {
```
Comment (skqr) [inline Util/RepositoryHelper.php]: This should be implemented as a filter, just like the default filter. Maybe `SortFilter`?

Comment (chebbix) [inline Util/RepositoryHelper.php]: Sort is not a Filter

File: Util/RepositoryHelper.php (hunk 1 above)
Comment (chebbix) [inline Util/RepositoryHelper.php]: Cipayo!

**Comment by skqr** [comment]
Not bad. Not bad at all. Once the tests are passing, I would advice merging.

**Comment by skqr** [comment]
Yeah.

ron

**Pull request merged**